common/src/main/cpp/extensions/openxr_fb_composition_layer_settings_extension_wrapper.cpp

```diff
@@ -153,12 +165,31 @@ TypedArray<Dictionary> OpenXRFbCompositionLayerSettingsExtensionWrapper::_get_vi
 		properties.push_back(sharpening_mode);
 	}
 
+	{
+		Dictionary enable_auto_filter;
+		enable_auto_filter["name"] = ENABLE_AUTO_FILTER_PROPERTY_NAME;
+		enable_auto_filter["type"] = Variant::BOOL;
+		enable_auto_filter["hint"] = PROPERTY_HINT_NONE;
+		properties.push_back(enable_auto_filter);
+	}
+
+	{
+		Dictionary auto_options;
+		auto_options["name"] = AUTO_OPTIONS_PROPERTY_NAME;
+		auto_options["type"] = Variant::INT;
+		auto_options["hint"] = PROPERTY_HINT_FLAGS;
+		auto_options["hint_string"] = "Normal Supersampling,Quality Supersampling,Normal Sharpening,Quality Sharpening";
```

Comment:
What happens if both the `normal` and `quality` option variants are selected?

Reply:
This is a good question, I've been thinking that it could swap between all possibilities, but the spec isn't really explicit on this.

Main issue is I'm not able to visually distinguish when a normal filter vs. a quality one is being applied. But whether just one or both of the options are selected, I'm definitely able to notice the filters swap from disabled to enabled, and I don't see why the filter wouldn't automatically swap between normal and quality as well. I just don't have a way of knowing which of the two is being applied.

The one bit from the spec on passing options:
> A candidate pool of preferred layer filtering methods from [XrCompositionLayerSettingsFlagBitsFB](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XrCompositionLayerSettingsFlagBitsFB) must be passed along with XR_COMPOSITION_LAYER_SETTINGS_AUTO_LAYER_FILTER_BIT_META. The runtime may apply the appropriate filter when rendering the layer.

Reply:
My reading of the spec is that if you give both `normal` and `quality` then it will include both of those in its list of potential options that auto could switch between.

But like Logan, when testing the "manual" settings, I can't tell visually when quality is enabled either.

Reading [this article](https://developer.oculus.com/documentation/native/android/mobile-openxr-composition-layer-filtering/), it makes it sound like the `quality` options are only a little better visually, but take a lot more resources.

Reply:
From the [Enable Layer Filtering](https://developer.oculus.com/documentation/native/android/mobile-openxr-composition-layer-filtering/#enable-layer-filtering) section, they mention the followings:
> If both bits are set, the compositor defaults to the normal supersampling method.

> If both bits are set, the compositor defaults to the normal sharpening method.

So the current approach should be good to go!
Let me know if I should go ahead and merge the changes.

Reply:
@devloglogan Let me know if I can go ahead and merge this PR.